### ISSUE TITLE
PPL Weight file

### ISSUE THREAD

**Issue by seymayucer** [cryu854/StyleGAN2#3]
Hello, the link for the model of perceptual metric is not working can you please update it?

 https://drive.google.com/u/2/uc?id=12qqzTA2HGM50Qnt12FYyjgL6p4nl5bNZ&export=download

 _tf.keras.utils.get_file( 'lpips.h5','https://drive.google.com/u/2/uc?id=12qqzTA2HGM50Qnt12FYyjgL6p4nl5bNZ&export=download',cache_subdir='weights-lpips')   # Store at ~/.keras/weights-lpips_

Thanks

**Comment by jennyyachen** [comment]
same problem, or could you show us where to download the file?

Thanks a lot
